compiler/src/dotty/tools/dotc/transform/ExplicitOuter.scala

```diff
@@ -355,14 +355,21 @@ object ExplicitOuter {
     }
 
     /** The path of outer accessors that references `toCls.this` starting from
-     *  the context owner's this node.
+     *  node `start`, which defaults to the context owner's this node.
+     *  @param  outOfContext  When true, we take the `path` in code that has been inlined
+     *                        from somewhere else. In that case, we need to stop not
```

Comment:
But we always have `outOfContext = true` in `transformSelect`, not just in code that has been inlined.
Shouldn't the condition be something like `!tpd.enclosingInlineds.isEmpty` instead?